### PR TITLE
Missing entity_id in automation recipe

### PR DESCRIPTION
**Pull request by dasdundave**
## Proposed change

Added missing entity_id in automation recipe

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @davidw3591, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23112).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/calendar.markdown [hunk 1]
```diff
@@ -99,8 +99,10 @@ automation:
         - "{{ trigger.event == 'start' }}"
       then:
         - service: light.turn_on
+          entity_id: light.front
       else:
         - service: light.turn_off
+          entity_id: light.front
```
Comment (frenck) [inline source/_integrations/calendar.markdown:105]: As this is an if/else, maybe we should use a different entity in the else clause :)

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/calendar.markdown (hunk 1 above)
Comment (dasdundave) [reply source/_integrations/calendar.markdown:105]: I don't see why?

```
Example: Calendar Event Light Schedule
This example consists of:

For the calendar entity calendar.device_automation
When event summary contains Front Lights
Turn on and off light named light.front when the event starts and ends
```

Comment (frenck) [reply source/_integrations/calendar.markdown:105]: LOL 🤦 

Sorry by bad, the service calls are different, one call turns it on, the other turns it off missed that 🙈 

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @davidw3591 👍

**Pull request merged**